doc/whats_new/v0.22.rst

```diff
@@ -19,6 +19,8 @@ random sampling procedures.
 
 - :class:`decomposition.SparsePCA` where `normalize_components` has no effect
   due to deprecation.
+..
+    TO FILL IN AS WE GO
```

Comment:
This is probably due to a merge conflict and should be removed.

Reply:
thank you for catching that, I've removed those two lines.